### DISCUSSION TITLE
how to change the vertical distance between nodes

### DISCUSSION THREAD

**Discussion by graemep-nz** [xrr2016/flutter-tree#16] [General]
Is it possible to change/ reduce the vertical distance between nodes?

Also in the online example
https://fluttertree.coldstone.fun/#/
there is a spinning mouse cursor.
Is that spinning because there is a real delay or is it just for demo purposes?

**Comment by xrr2016** [comment]
> 

You could fork this repo and custom vertical distance between nodes;
Is real delay show get data from server, but in the example just `await Future.delayed(const Duration(seconds: 1));`
